README.md

```diff
@@ -1,13 +1,15 @@
 # fgit
-A tool to replace your github URL to fastgit URL
+A tool to replace your GitHub URL with fastgit URL automatically when cloning repositories.
 
-## How To Use？
+[简体中文](./README-zh_CN.md)
+
+## How To Use?
```

Comment:
MD022/blanks-around-headings/blanks-around-headers: Headings should be surrounded by blank lines 
See more [here](https://github.com/updownpress/markdown-lint/blob/master/rules/022-blanks-around-headers.md).